lib/streamlit/delta_generator.py

```diff
@@ -426,12 +449,7 @@ def expander(self, label=None, expanded=False):
         return self._block(block_proto=block_proto)
 
     def favicon(
```

Comment:
I really don't know why this keeps flopping back and forth. In any case, favicon() is an unused method; I'll kill it soonish